tests/system_tests/test_core/test_console.py

```diff
@@ -55,7 +89,7 @@ def test_offline_compression(user, capfd, console):
 
             print("\x1b[A\r\x1b[J\x1b[A\r\x1b[1J")  # noqa: T201
 
-            time.sleep(2)
+        _wait_for_legacy_service_transaction_log()
```

Comment:
should we have this logic conditional or just keep it as is and remove it once we remove the legacy service?

Reply:
i think it is fine as is... ignore my previous comment